### ISSUE TITLE
rack-protection v3.0.0: LoadError: cannot load such file -- rack-protection

### ISSUE THREAD

**Issue by nbibler** [sinatra/sinatra#1817]
The recent [rack-protection v3.0.0 release](https://rubygems.org/gems/rack-protection/versions/3.0.0) appears to fail to load:

```
Run bundle exec rails db:prepare
rails aborted!
LoadError: cannot load such file -- rack-protection
```

In this case, it's being used in an application as a sub-dependency of [`omniauth` v2.1.0](https://rubygems.org/gems/omniauth/versions/2.1.0) and not directly used within the hosting application.

```
% git grep rack-protection
Gemfile.lock:      rack-protection
Gemfile.lock:    rack-protection (3.0.0)
```

**Comment by nbibler** [comment]
This _may_ (although I'm not sure exactly how) have something to do with [a commit 4 hours ago](b2287409ecf78efd81c80c7c0526d00e755ca4a9) which also appears to be failing CI.

**Comment by nbibler** [comment]
This is likely related to https://github.com/omniauth/omniauth/issues/1089.

**Comment by olleolleolle** [comment]
@nbibler Thanks for reporting, but the unreleased https://github.com/sinatra/sinatra/commit/b2287409ecf78efd81c80c7c0526d00e755ca4a9 commit would be what could alleviate that, when it gets into a release.

Until such a time, the `omniauth` gem could change its require away from the `rack-protection`.

**Comment by JonathanPreminger** [comment]
+1

**Comment by jkowens** [comment]
Fix released in v3.01.

**Comment by nbibler** [comment]
Thank you! 👏
